app/controllers/external_users/claims_controller.rb

```diff
@@ -112,7 +113,8 @@ def unarchive
   def new
     load_offences_and_case_types
     build_nested_resources
-    track_visit({ url: 'external_user/%{type}/claim/new/page1', title: 'New %{type} claim page 1' }, claim_tracking_substitutions)
+    track_visit({ url: 'external_user/%{type}/claim/new/page1',
+                  title: 'New %{type} claim page 1' }, claim_tracking_substitutions)
```

Comment:
i'd put another linebreak before `claim_tracking_substitutions` for consistency with code below